src/Umbraco.Core/PropertyEditors/PropertyEditorValueConvertersResolver.cs

```diff
@@ -8,6 +8,7 @@ namespace Umbraco.Core.PropertyEditors
     /// <summary>
 	/// Manages the list of IPropertyEditorValueConverter's
     /// </summary>
+    [Obsolete("IPropertyValueConverter is obsolete, but we need to support them until v8, to resolve the correct converter types use: PropertyValueConvertersResolver")]

```

Comment:
`IPropertyEditorValueConverter` is the obsolete one, whoops, will change the text